engine/src/main/java/org/terasology/rendering/nui/layers/ingame/PauseMenu.java

```diff
@@ -33,7 +33,7 @@ public void initialise() {
         WidgetUtil.trySubscribe(this, "settings", widget -> getManager().pushScreen("settingsMenuScreen"));
         WidgetUtil.trySubscribe(this, "mainMenu", widget -> CoreRegistry.get(GameEngine.class).changeState(new StateMainMenu()));
         WidgetUtil.trySubscribe(this, "exit", widget -> CoreRegistry.get(GameEngine.class).shutdown());
-        WidgetUtil.trySubscribe(this, "crashReporter", widget -> CrashReporter.report(new Throwable("Report an error."), LoggingContext.getLoggingPath()));
+//        WidgetUtil.trySubscribe(this, "crashReporter", widget -> CrashReporter.report(new Throwable("Report an error."), LoggingContext.getLoggingPath()));
```

Comment:
Might have pulled latest code without doing a fresh `gradlew idea` ? The CR dependency was recently moved around a bit (from PC facade back to engine) so it could be hit from the main menu screen. 

That would fit the symptom pretty perfectly so hopefully that's it :-)